api.py

```
@@ -49,6 +48,7 @@ def get_api_key(db: Session = Depends(get_db), api_key: str = Security(api_key_c
 async def api_add_swimmer(
     request: Request,
     db: Session = Depends(get_db),
+    scraper: SwimrankingsScraper = Depends(get_scraper),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)